packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -581,7 +581,7 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
       statusDiv.setAttribute('role', 'status')
       statusDiv.setAttribute('aria-live', 'polite')
       statusDiv.setAttribute('aria-relevant', 'additions text')
-      Object.assign(statusDiv.style, screenReaderContainerStyles)
+      _.assign(statusDiv.style, screenReaderContainerStyles)
```

Comment:
https://lodash.com/docs/4.17.11#assign
> Note: This method mutates object and is loosely based on Object.assign.

Reply:
would we need to ensure that `Object.assign` won't be used anywhere in the codebase? I would rather expect that we will introduce polyfill that will ensure that even if `Object.assign` is used somewhere in the codebase, it will be handled correctly. Currently we are just introducing implicit rule of using `_.assign` vs `Object.assign` that will just serve as short-term patch.

I do understand that we may introduce lint rules, but for this particular case it seems that much easier solution will be just to introduce a polyfill - this will ensure that all the previous code would work without any additional changes necessary

Reply:
I support too adding polyfill, it will be better for the developers. And yep, we can add additional polyfills if there are other constraints in the future.